apps/ledger-live-mobile/android/build.gradle

```diff
@@ -16,15 +16,15 @@ buildscript {
             ndkVersion = "24.0.8215888"
         } else {
             // Otherwise we default to the side-by-side NDK version from AGP.
-            ndkVersion = "21.4.7075529"
+            ndkVersion = "23.1.7779620"
         }
     }
     repositories {
         google()
         mavenCentral()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:4.2.2")
+        classpath("com.android.tools.build:gradle:7.4.1")
```

Comment:
Sounds like a huge upgrade (v4 -> v7), maybe we don't want to handle this on our own only since this could perhaps have other consequences outside the scope of this team ?